haskell-language-server.cabal

```diff
@@ -366,13 +360,12 @@ common brittany
     build-depends: hls-brittany-plugin ^>= 1.0
     cpp-options: -Dhls_brittany
 
-executable haskell-language-server
+library plugins
```

Comment:
this might make it impossible to use stack for loading HLS into HLS. I.e. HLS needs to be configured to use cabal for development with this change due to https://github.com/commercialhaskell/stack/issues/4564

Reply:
Finally time to drop the stack descriptors and the Circle CI